VideoAPI/VideoApi.AcceptanceTests/Features/Events.feature

```diff
@@ -5,7 +5,25 @@
 
 Scenario: Add conference event
 	Given I have a conference
-    And I have a valid conference event request for event type Joined
-    When I send the request to the endpoint
-    Then the response should have the status NoContent and success status True
+	And I have a valid conference event request for event type Joined
+	When I send the request to the endpoint
+	Then the response should have the status NoContent and success status True
 	And the status is updated
+
+Scenario: Add participant event
+	Given I have a conference
+	Given I have a valid conference event request for event type <EventType>
+	When I send the request to the endpoint
+	Then the response should have the status NoContent and success status True
+
+	Examples:
+		| EventType             |
+		| Joined                |
+		| Disconnected          |
+		| Pause                 |
+		| Close                 |
+		| Leave                 |
+		| JudgeAvailable        |
+		| JudgeUnavailable      |
```

Comment:
this line should be the only change

Reply:
Need to introduce standard, in my case VS2019 use default to format it

Reply:
Sorry Shaed, but I could not remove these white space now, it's difficult as I dont't know how it was before and VS does not show it. I think it's not so important at the moment. But next time I will adjust the indent and tab in my VS to be agreed with the standard we set.